### PR TITLE
Pridanie endpointu projects-users

### PR DESCRIPTION
**Pull request by fejjo**
Main PR: https://github.com/keboola/connection/pull/2276
JIRA Issue: https://keboola.atlassian.net/browse/KBC-31

### CODE REVIEW

File: tests/ListProjectsUsersTest.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        ]);
+
+        $this->normalUserClient->createProject($this->organization['id'], [
+            'name' => 'Test Project',
+        ]);
+
+        $this->normalUserClient->removeUserFromOrganization($this->organization['id'], $this->normalUser['id']);
+
+        $this->expectException(ClientException::class);
+//        @todo You don't have access to the organization
```
Comment (fejjo) [inline tests/ListProjectsUsersTest.php]: Tu dostavam tiez inu hlasku bud robim nieco blbo alebo to bude ako v tom dalsom teste

File: tests/ListProjectsUsersTest.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Keboola\ManageApiTest;
+
+use Keboola\ManageApi\ClientException;
+
+class ListProjectsUsersTest extends ClientTestCase
+{
+    const LIST_ORGANIZATION_PROJECTS_USERS_EXCEPTION_MESSAGE = 'Only organization members can get list of projects users';
```
[the remark this answers is not in the document]
Comment (ErikZigo) [reply tests/ListProjectsUsersTest.php]: To by ti hlásit neměl

https://github.com/keboola/kbc-manage-api-php-client/blob/7a6b804f78160352581b807425ba65fceeabdb44/composer.json#L4-L7

A Dockerfile je taky na PHP 7.1.

Na PHP 5.6 je jen Storage API client

File: tests/ListProjectsUsersTest.php (hunk 1 above)
Comment (fejjo) [reply tests/ListProjectsUsersTest.php]: Tak som oddelal to todo ma to byt tak ako je to v tom dalsom teste.

**Comment by fejjo** [comment]
Test som pridal.

**Pull request merged**